projects/1.20/assets/enderite-cj-edition/enderite/lang/zh_cn.json

```
@@ -0,0 +1,113 @@
+{
+  "creativemodetab.enderite.enderite_tab": "末影合金",
+
+  "enchantment.enderite.night_vision": "夜视",
+  "enchantment.enderite.no_gravity": "失重",
+  "enchantment.enderite.void_floating": "虚空飘浮",
+  "enchantment.enderite.night_vision.desc": "给予玩家夜视效果",
+  "enchantment.enderite.no_gravity.desc": "赋予物品无重力",
+  "enchantment.enderite.void_floating.desc": "保护物品不受虚空伤害",
```

```suggestion
  "enchantment.enderite.void_floating.desc": "保护物品不被虚空吞噬",
```
库里应该有这条，建议比对一下
但是虚空理论上是不会对物品造成伤害的